src/formatDate.js

```diff
@@ -7,8 +7,58 @@
  *
  * @returns {string}
  */
+
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromSeperator = fromFormat[fromFormat.length - 1];
```

Comment:
The variable name 'fromSeperator' contains a typo; it should be 'fromSeparator'. This helps maintain correct spelling throughout the codebase.